feature-configs/cn-ran-overlays/ran-profile/is_ready.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Check for updated worker-cnf node
+
+oc wait mcp/worker-cnf --for condition=updated --timeout 1s
+
+
+# Check for real-time kernel.
+
+WORKER_NODE=$(oc get nodes --selector="node-role.kubernetes.io/worker-cnf" --output="name")
+
+echo "worker-cnf node is: ${WORKER_NODE}"
```

Comment:
there might be more than one worker in that pool

Reply:
I have removed this check altogether and will move it to the QE pipeline, where it will be accompanied by other checks & tests. It wasn't necessary here.